moz_sql_parser/templates/macros.jinja2

```diff
@@ -0,0 +1,227 @@
+{% macro block( content ) %}
```

Comment:
This will take me longer to review.  I have not used jinja before: I would like to know if we can pretty print the SQL in the future using this tool.  I am dubious of the conditional code in the template, it may be easier in Python directly.